be/src/script/script.cpp

```diff
@@ -119,6 +121,41 @@ static int64_t unix_seconds() {
     return UnixSeconds();
 }
 
+static std::string exec(const std::string& cmd) {
+    std::string ret;
+
+    FILE* fp = popen(cmd.c_str(), "r");
+    if (fp == NULL) {
+        ret = strings::Substitute("popen failed: $0 cmd: ", strerror(errno), cmd);
+        return ret;
+    }
+
+    char buff[4096];
+    while (true) {
+        size_t r = fread(buff, 1, 4096, fp);
+        if (r == 0) {
+            break;
+        }
+        ret.append(buff, r);
+    }
+    int status = pclose(fp);
+    if (status == -1) {
+        ret.append("pclose failed");
+    } else if (status != 0) {
+        ret.append(strings::Substitute("exit: $0", status));
```

Comment:
```suggestion
        ret.append(strings::Substitute("exit: $0", strerror(errno));
```

Reply:
`status` is subprocess's exit code, it's not an error code that strerror can handle.